raspiaudio-muse-luxe.yaml

```
@@ -108,22 +166,62 @@ voice_assistant:
 sensor:
   - platform: adc
     pin: GPIO33
-    name: Battery
-    icon: "mdi:battery-outline"
+    name: Battery voltage
     device_class: voltage
+    unit_of_measurement: "V"
+    accuracy_decimals: 2
     state_class: measurement
     entity_category: diagnostic
-    unit_of_measurement: V
     update_interval: 15s
-    accuracy_decimals: 3
-    attenuation: 11db
-    raw: true
+    attenuation: auto
     filters:
-      - multiply: 0.00173913 # 2300 -> 4, for attenuation 11db, based on Olivier's code
+      - multiply: 2 # https://forum.raspiaudio.com/t/esp-muse-luxe-bluetooth-speaker/294/12
       - exponential_moving_average:
           alpha: 0.2
           send_every: 2
       - delta: 0.002
+    on_value:
+      then:
+        - sensor.template.publish:
+            id: battery_percent
+            state: !lambda 'return x;'
+
+  - platform: template
+    name: Battery
+    id: battery_percent
+    device_class: battery
+    unit_of_measurement: "%"
+    accuracy_decimals: 0
+    state_class: measurement
+    entity_category: diagnostic
+    update_interval: 15s
```

This sensor is published to, it doesnt need to update itself
```suggestion
    update_interval: never
```